File: examples/csv/01.read-csv.py
#! /usr/bin/env python

import re

for line in open("data.csv"):
    if line.startswith("#"):
        continue

    data = line.split(",", 9)

    if re.search("SALTA|CORDOBA", data[2]):
        continue

    mujeres, varones = int(data[3]), int(data[4])

    if mujeres > varones:
        nombres = data[0:3]
        nombres.reverse()
        print ", ".join(nombres)

File: examples/csv/01.read-csv.rb
#! /usr/bin/env ruby

File.open("data.csv").each do |line|
  next if line.start_with? "#"

  data = line.split(",", 9)

  next if data[2] =~ /SALTA|CORDOBA/

  mujeres, varones = data[3].to_i, data[4].to_i

  puts data.first(3).reverse.join(", ") unless varones > mujeres
end
